dataline-integrations/singer/stripe/source/entrypoint.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+# TODO: replace with more general version
+# hacky version that expects ordering of args "--discover --config config.json"
```

Comment:
Is this comment backwards? is it supposed to be `--config config.json --discover`?

Reply:
fixed